File: CNC modeling 2.0/src/main/java/com/sergey/pisarev/controller/AboutController.java
package com.sergey.pisarev.controller;

import java.awt.*;
import java.io.IOException;
import java.net.URISyntaxException;
import java.net.URL;

public class AboutController {

    public void onAction() {
        try {
            Desktop.getDesktop().browse(new URL("https://github.com/pisarevS").toURI());
        } catch (IOException | URISyntaxException e) {
            e.printStackTrace();
        }
    }
}

File: CNC modeling 2.0/src/main/java/com/sergey/pisarev/controller/AboutController.kt
package com.sergey.pisarev.controller

import java.awt.Desktop
import java.io.IOException
import java.net.URISyntaxException
import java.net.URL

class AboutController {
    fun onAction() {
        try {
            Desktop.getDesktop().browse(URL("https://github.com/pisarevS").toURI())
        } catch (e: IOException) {
            e.printStackTrace()
        } catch (e: URISyntaxException) {
            e.printStackTrace()
        }
    }
}